Build.fsx

```diff
@@ -126,9 +125,9 @@ Target "TestOnly" (fun _ ->
                                                 ResultSpecs = ["NUnit3TestResult.xml;format=nunit2"] })
 )
 
-Target "BuildAndTestOnly" (fun _ -> ())
-Target "Build" (fun _ -> ())
-Target "Test"  (fun _ -> ())
+Target "BuildAndTestOnly" DoNothing
+Target "Build" DoNothing
+Target "Test"  DoNothing
```

Comment:
I think it looks rather wasteful to have all these no-op targets laying around. Couldn't we remove them?